### ISSUE TITLE
ModifierMissing rule does not sees GlanceModifier

### ISSUE THREAD

**Issue by phansier** [twitter/compose-rules#51]
**Describe the bug**
ModifierMissing gives false positive alert for GlanceAppWidget

**To Reproduce**
```
class FirstGlanceWidget : GlanceAppWidget {
...
    @Composable
    fun SmallWidget(
        modifier: GlanceModifier = GlanceModifier,
    )
...
}
```

**Expected behavior**
No alert

**Comment by mrmans0n** [comment]
Interesting, we haven't used that one internally yet. I will add support so that is correctly used for all modifier related rules (missing, ordering, etc).
